nuitka/tools/quality/auto_format/YamlFormatter.py

```diff
@@ -355,15 +366,28 @@ def formatYaml(path):
                 )
 
         if "implicit-imports" in sorted_entry:
-            sorted_entry["implicit-imports"] = dict(
-                OrderedDict(
-                    [
-                        (key, entry["implicit-imports"][key])
-                        for key in IMPLICIT_IMPORTS_KEYS
-                        if key in entry["implicit-imports"].keys()
-                    ]
+            for sub_entry in sorted_entry["implicit-imports"]:
+                sub_entry = dict(
```

Comment:
The `dict(OrderedDict` calls look strange, is this because Yaml will not take them back unless as dict